### PR TITLE
Removes songs that suck

### PR DESCRIPTION
**Pull request by swissloaf**
### Intent of your Pull Request

Removes Jojo Theme, Running in 90s, Danzig, Elbow, and some other one idk.

#### Changelog

:cl:   
rscdel: Removes Jojo Theme, Running in 90s, Danzig, Elbow, and some other one idk.
/:cl:

### CODE REVIEW

**Review by nesisknesis — APPROVED** [review]
i approve

**Comment by overlord12345** [comment]
dont remove jojo theme

**Comment by Amelia0010** [comment]
```
/datum/controller/subsystem/ticker/proc/choose_lobby_music()
	//Add/remove songs from this list individually, rather than multiple at once. This makes it easier to judge PRs that change the list, since PRs that change it up heavily are less likely to meet broad support
	//Add a comment after the song link in the format [Artist - Name]
```

**Comment by swissloaf** [comment]
> ```
> /datum/controller/subsystem/ticker/proc/choose_lobby_music()
> 	//Add/remove songs from this list individually, rather than multiple at once. This makes it easier to judge PRs that change the list, since PRs that change it up heavily are less likely to meet broad support
> 	//Add a comment after the song link in the format [Artist - Name]
> ```

cool

**Comment by deathslayer2481** [comment]
Why would you remove when you can fix

**Comment by ynot01** [comment]
dont remove jojo theme

File: yogstation/code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -20,14 +20,12 @@
 		"https://www.youtube.com/watch?v=GISnTECX8Eg",						// Chris Remo - Space Asshole
 		"https://www.youtube.com/watch?v=w34fSnJNP-4",						// Leslie Fish - Dawson's Chrstian (Singer unknown)
 		"https://www.youtube.com/watch?v=le1eD6I7k4s",						// Ronald Jenkee - Piano Wire
-		"https://www.youtube.com/watch?v=2MtOpB5LlUA",						// Yugo Kanno - il vento d'oro (Golden Wind)
-		"https://www.youtube.com/watch?v=Uhy_e5wIX44",						// Maurizio De Jorio - Running in the 90s (Remix by Sytricka)
 		"https://www.youtube.com/watch?v=na_Zac23b5E",						// Basil Poledouris - Klendathu Drop
 		"https://www.youtube.com/watch?v=WcLzUZSGT6Q",						// Hans Zimmer, Benjamin Wallfisch - Mesa
 		"https://www.youtube.com/watch?v=Aj0htz_RzlY",						// Mega Drive - Converter
+		"https://www.youtube.com/watch?v=zlD5uvOt_Sw",                      // HL2 - Apprehension and Evasion
```
Comment (Amelia0010) [inline yogstation/code/controllers/subsystem/ticker.dm:26]:
```suggestion
		"https://www.youtube.com/watch?v=zlD5uvOt_Sw",                      // Kelly Bailey - Apprehension and Evasion
```

File: yogstation/code/controllers/subsystem/ticker.dm [hunk 2]
```diff
@@ -20,14 +20,12 @@
...
 		"https://www.youtube.com/watch?v=d2xkpz-26jM",						// Admiral Hippie - Clown.wmv
-	    "https://www.youtube.com/watch?v=UlHGGKgzgzI",                      // Elbow - Leaders of the Free World
-		"https://www.youtube.com/watch?v=02FVRMhKR9I",                      // Danzig - How The Gods Kill
+		"https://www.youtube.com/watch?v=KFQpSp-3APw",                      // SS13 - Spectacular Suns
```
Comment (Amelia0010) [inline yogstation/code/controllers/subsystem/ticker.dm:28]:
```suggestion
		"https://www.youtube.com/watch?v=KFQpSp-3APw",                      // Andreas Pflüger & David Hasselhoff - Pingu Opening Sequence (Season 3)
```

**Review by Amelia0010 — CHANGES_REQUESTED** [review]
How do you manage to break both of the rules for editing the list?

File: yogstation/code/controllers/subsystem/ticker.dm (hunk 2 above)
Comment (AsV9) [reply yogstation/code/controllers/subsystem/ticker.dm:28]: nice one

**Comment by AsV9** [comment]
atomize the PR

**Pull request closed without merging**